infrastructure/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:8-jdk-alpine3.8
```

Comment:
Future Optimization if required:
The docker image size can be further reduced by using only the JRE rather than the JDK.

Reply:
for now, it's preferred having jdk. Agree, we can migrate to JRE image if needed